File: ANSv4/Catalogo3/classes/InterDepositoFacturas.vb
Option Explicit On

Public Class InterDepositoFacturas

    ' Define como se llama este modulo para el control de errores
    Private Const m_sMODULENAME_ As String = "clsInterDepositoFacturas"

    Private mvarT_Comprobante As String
    Private mvarN_Comprobante As String
    Private mvarImporte As Single

    Public Property T_Comprobante() As String
        Get
            Return mvarT_Comprobante
        End Get
        Set(ByVal value As String)
            mvarT_Comprobante = value
        End Set
    End Property

    Public Property N_Comprobante() As String
        Get
            Return mvarN_Comprobante
        End Get
        Set(ByVal value As String)
            mvarN_Comprobante = value
        End Set
    End Property

    Public Property Importe() As Single
        Set(ByVal value As Single)
            mvarImporte = value
        End Set
        Get
            Return mvarImporte
        End Get
    End Property


End Class

File: ANSv4_r1/Catalogo4/_interdeposito/InterDepositoFacturas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Catalogo._interdeposito
{
    public class InterDepositoFacturas
    {
        private string mvarT_Comprobante;
        private string mvarN_Comprobante;
        private float mvarImporte;

        public string T_Comprobante
        {
            get { return mvarT_Comprobante; }
            set { mvarT_Comprobante = value; }
        }

        public string N_Comprobante
        {
            get { return mvarN_Comprobante; }
            set { mvarN_Comprobante = value; }
        }

        public float Importe
        {
            get { return mvarImporte; }
            set { mvarImporte = value; }
        }
    }
}